lib/src/presentations/verify_email/screen/verify_email_view.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:jobspot/src/core/config/localization/app_local.dart';
+import 'package:jobspot/src/presentations/verify_email/cubit/verify_email_cubit.dart';
+import 'package:jobspot/src/presentations/verify_email/domain/router/verify_email_coordinator.dart';
+
+import '../../../core/constants/constants.dart';
```

Comment:
Should be import full path instead of ".../.../.."

Reply:
I fixed it